internal/pkg/service/buffer/storage/local/writer/config.go

```diff
@@ -12,15 +12,21 @@ type config struct {
 	writerFactory Factory
 	// fileOpener provides file opening, a custom implementation can be useful for tests.
 	fileOpener FileOpener
+	// watchDrainFile activates watching for drainFile changes (creation/deletion),
+	// otherwise the file is checked only on the volume opening.
+	// Default Linux OS limit is 128 inotify watchers = 128 volumes.
+	// The value is sufficient for production but insufficient parallel for tests.
+	watchDrainFile bool
 }
```

Comment:
V Linuxe mozes mat by default maximalne 128 FS notifiers/watchers, ... je to dost pre produkciu ale v testoch, ktorych je viac ako `128` a bezia paralelne to musim vypnut.